src/components/applicant/ApplicantInfo.tsx

```diff
@@ -1,46 +1,39 @@
 import React from 'react';
-import DashboardListCard from '../components/dashboard/dashboard-list/DashboardListCard';
-import ApplicantInfoContent from '../components/applicant/ApplicantInfoContent';
-import SideBar from '../components/sidebar/SideBar';
-import ApplicantInfoHeader from '../components/applicant/ApplicantInfoHeader';
+import DashboardListCard from '../dashboard/dashboard-list/DashboardListCard';
+import ApplicantInfoContent from './ApplicantInfoContent';
+import ApplicantInfoHeader from './ApplicantInfoHeader';
 
 interface IApplicantInfoProps {
     name: string;
     role: string;
 }
 
-const ApplicantInfo: React.FunctionComponent = () => {
-    // sample applicantInfo list to test the prev/next applicant feature
-    const applicantList: IApplicantInfoProps[] = [
-        { name: 'John Doe', role: 'Developer Applicant' },
-        { name: 'Selene Dion', role: 'Developer Applicant' },
-        { name: 'Happy Holland', role: 'Designer Applicant' },
-        { name: 'Lionel Ronaldo', role: 'Developer Applicant' },
-        { name: 'Tom Downey', role: 'Designer Applicant' },
-        { name: 'Donald Biden', role: 'Developer Applicant' },
-        { name: 'Fizz Buzz', role: 'Developer Applicant' },
-        { name: 'Dude Dude Bar', role: 'Designer Applicant' },
-        { name: 'Yeet Feet', role: 'Developer Applicant' },
-        { name: 'Paul Doll', role: 'Designer Applicant' },
-        { name: 'Shiloh Dynasty', role: 'Developer Applicant' },
-        { name: 'Mozart Beethoven', role: 'Designer Applicant' },
-        { name: 'Harin Wu', role: 'Developer Applicant' },
-        { name: 'Loot Toot', role: 'Designer Applicant' },
-        { name: 'Cringe Fest', role: 'Developer Applicant' },
-        { name: 'Lo Fi', role: 'Designer Applicant' },
-        { name: 'Hip Hop', role: 'Developer Applicant' },
-    ];
-    const [count, setCount] = React.useState(0);
-    return (
-        <div className="section view">
+class ApplicantInfo extends React.Component<{
+    applicantList: IApplicantInfoProps[];
+    applicant: { name: string; role: string };
+    count: number;
+    setCount: (newCount: number) => void;
+    viewDashboard: () => void;
+    viewApplicant: (newCount: number) => void;
+}> {
```

Comment:
Were we going to handle this in the dashboard? I think that ApplicantInfo can just take a single applicant (IApplicantInfoProps) which will probably simplify the component a lot. This may mean that we don't use hooks anymore and just standard functions in the Dashboard class.

Reply:
Passing in these props from Dashboard.tsx enables the Next Applicant, Prev Applicant, and Return to Dashboard navigation. Is there a cleaner way of doing this?

Reply:
Yeah that makes sense for the callbacks, I think the only difference was to just pass a singular applicant in instead of the whole list.

This applicant would be referenced in dashboard as applicantList[count]

Reply:
Oh my mistake! I think I missed that, sorry, I've pushed my revised code.